rerun-failed-run/action.yml

```diff
@@ -52,17 +52,53 @@ runs:
       with:
         app-id: ${{ steps.vault-secrets.outputs.RETRIGGER_APP_ID }}
         private-key: ${{ steps.vault-secrets.outputs.RETRIGGER_APP_KEY }}
-        owner: "camunda"
-        repositories: "infra-global-github-actions"
 
-    - name: Retrigger run
+    - name: Get repository name and owner
+      id: repository
       env:
-        GH_DEBUG: api
-        GITHUB_TOKEN: ${{ steps.github-token.outputs.token }}
+        REPOSITORY: ${{ inputs.repository }}
+      run: |
+        echo "owner=${REPOSITORY%/*}" | tee -a "$GITHUB_OUTPUT"
+        echo "name=${REPOSITORY##*/}" | tee -a "$GITHUB_OUTPUT"
       shell: bash
+
+    - name: Convert error messages list to JSON
+      id: error-messages
+      env:
+        ERROR_MESSAGES: ${{ inputs.error-messages }}
       run: |
-        error_messages="${{ inputs.error-messages }}"
-        result=$(echo "$error_messages" | jq -c -n -R '[inputs | select(length > 0)]')
-        echo "result: $result"
+        messages_json=$(
+          echo "${ERROR_MESSAGES}" | \
+            jq -c -n -R '[inputs | select(length > 0)]' | \
+            jq -R .
+        )
+        echo json="${messages_json}" | tee -a $GITHUB_OUTPUT
+      shell: bash
 
-        gh workflow run rerun-failed-run.yml -R camunda/infra-global-github-actions --ref=main -F run_id=${{ inputs.run-id }} -F repository=${{ inputs.repository }} -F error_messages="$result" -F notify_back_on_error=${{ inputs.notify-back-on-error }} -F notify_back_git_ref=${{ github.ref_name }}
+    - name: Trigger ${{ github.repository }}/retry-worfklow-run.yml
```

Comment:
I am not sure if I understand correctly, but here it looks like the original repository must implement `retry-workflow-run`. 

Are there cases where this is not `infra-global-github-actions`? If so, we should describe that in the accompanying readme I guess.

Reply:
Good catch! It should not be dynamic indeed, and always call camunda/infra-global-github-actions/retry-workflow-run.yml.
Since my test was on the same repo it worked and I did not spot that issue.

Reply:
Fixed.